src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task with its description and date of event.
+ */
+public class Event extends Task {
+    private LocalDate date;
+
+    /**
+     * Creates an Event task with its description and date of event.
+     * @param description description of event task.
+     * @param date date of event.
+     */
+    public Event(String description, LocalDate date) {
+        super(description, 'E');
+        this.date = date;
+    }
+
+    /**
+     * Gets the date of event of this task.
+     * @return Returns the date of event of this task.
+     */
+    public LocalDate getDate() {
+        return this.date;
+    }
+
+    /**
+     * Returns a formatted string of an Event task.
```

Comment:
You might have incorrectly formatted the javadocs for this method. 
Perhaps it should look something like this
```
/**
 * @returns a formatted string of an Event task. 
 */
```

Reply:
I apologise, this is slightly misleading! I should have documented both the description and the return value. Thanks for catching it!